lib/truemail.rb

```diff
@@ -29,12 +26,23 @@ def reset_configuration!
     end
 
     def validate(email, **options)
-      raise ConfigurationError, NOT_CONFIGURED unless configuration
+      raise_unless(configuration, NOT_CONFIGURED)
       Truemail::Validator.new(email, **options).run
     end
 
     def valid?(email, **options)
       validate(email, **options).result.valid?
     end
+
+    def host_audit
+      raise_unless(configuration, NOT_CONFIGURED)
+      Truemail::Auditor.new
```

Comment:
maybe will be better to have run class method. - Truemail::Auditor.run

Reply:
Yup!